hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightSearchEndpoint.java

```diff
@@ -386,4 +343,20 @@ private KeyEntityInfo createKeyEntityInfoFromOmKeyInfo(String dbKey,
     return keyEntityInfo;
   }
 
+  private boolean validateStartPrefixAndLimit(String startPrefix) {
```

Comment:
1. We are not validating limit here as method name says.
2. Similar method is duplicated in OMDBInsightEndPoint class also. Can we optimize here ?

Reply:
Made changes.